rootfs/api/serializers.py

```diff
@@ -337,6 +338,15 @@ def validate_healthcheck(self, data):
                 raise serializers.ValidationError(
                     "could not validate {}: {}".format(value, e.message))
 
+        # http://kubernetes.io/docs/api-reference/v1/definitions/#_v1_probe
```

Comment:
I looked and yes it is possible but then that would apply to both `readiness` and `liveness` probes.

Only `liveness` has this restriction, `readiness` appears not to have it